### ISSUE TITLE
Error on right click on transparent section with nothing

### ISSUE THREAD

**Issue by Ian-Hitt** [avocode/avocode#912]
IMPORTANT: Please try to describe how did it happen.

-----
App v.2.8.0
Source UID: 30049
```
TypeError: Cannot read property 'get' of null
    at e._onContextMenu (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:29:3953)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:28:31911
    at t.module.exports.Emitter.emit (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/emissary/lib/emitter.js:118:15)
    at t._handleContextMenu (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:23:22233)
    at HTMLDivElement.<anonymous> (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:23:16068)
    at HTMLDivElement.d (file:///Applications/Avocode.app/Contents/Resources/app.asar/static/raven.min.js:2:3637)
```

**Comment by vojtatranta** [comment]
Thanks!!
What have you been trying to do? We know that it was something with context menu, but can you explain what was it exactly?
Thanks!

**Comment by jakubzitny** [comment]
This will be fixed for `2.8.1`.

**Comment by jakubzitny** [comment]
Hey, we've just released [Avocode 2.8.1](http://avocode.com/download/) which solves this issue. Please, check it out and let us know if you have any more problems with it.

Thanks ;)
